bin/riak-bucket-exporter.js

```diff
@@ -111,7 +119,14 @@ function processKey(key, cb) {
     }
     fs.appendFileSync(program.file, JSON.stringify(out,null,'\t'));
     first=false;
-    return cb();
+    
+    if (deleteKeys) {
```

Comment:
I've made all your suggested changes.  I think it's cleaner that way.

Wasn't sure how you would increment the version number though.  I was tempted to increment it to 0.2.0 (minor increment since there's a new feature), but wanted to get your feedback before doing that.